sktime/forecasting/timesfm_forecaster.py

```diff
@@ -100,6 +100,9 @@ class TimesFMForecaster(_BaseGlobalForecaster):
         the installation of required packages manually. If False, the class will enforce
         the default dependencies required for the vendor library or the pypi
         package, as described above, via ``use_source_package``.
+    dl_engine: str, default="jax"
```

Comment:
You have also source engine. But it is not mentioned here.

Reply:
actually I tried not to mix `dl_engine` and `use_source_package`
```py
  if self.use_source_package:
      self.engine = _TimesFMSourceEngine(**self.engine_kwargs)
  elif self.dl_engine == "torch":
      self.engine = _TimesFMTorchEngine(**self.engine_kwargs)
  elif self.dl_engine == "jax":
      self.engine = _TimesFMJaxEngine(**self.engine_kwargs)
```

Also, `sktime` uses the previous version of `timesfm` for source package. I could not set up the latest version of `timesfm` on my system. However the latest version of timesfm, gives user the option to choose between `jax` and `torch`. This version was giving some bugs so I didnot update this in sktime.